packages/sanity/src/desk/comments/plugin/field/CommentFieldButton.tsx

```diff
@@ -114,7 +116,24 @@ export function CommentFieldButton(props: CommentFieldButtonProps) {
     onDiscard()
   }, [closePopover, onDiscard])
 
-  useClickOutside(startDiscard, [popoverElement])
+  const handlePopoverKeyDown = useCallback(
```

Comment:
Start discard comment not only when pressing "Escape" in the comment input. For example, if the user focus the "Send" button – we still want the user to be able to start discard the comment.